hello-ethernet/src/main.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(golioth_hello, LOG_LEVEL_DBG);
+
+#include <net/coap.h>
+#include <net/golioth/system_client.h>
+#include <net/golioth/wifi.h>
+
+#if IS_ENABLED(CONFIG_NET_L2_ETHERNET)
+#include <net/net_if.h>
+#endif
+
+static struct golioth_client *client = GOLIOTH_SYSTEM_CLIENT_GET();
+
+static void golioth_on_message(struct golioth_client *client,
+			       struct coap_packet *rx)
+{
+	uint16_t payload_len;
+	const uint8_t *payload;
+	uint8_t type;
+
+	type = coap_header_get_type(rx);
+	payload = coap_packet_get_payload(rx, &payload_len);
+
+	if (!IS_ENABLED(CONFIG_LOG_BACKEND_GOLIOTH) && payload) {
+		LOG_HEXDUMP_DBG(payload, payload_len, "Payload");
+	}
+}
+
+void main(void)
+{
+	int counter = 0;
+	int err;
+
+	LOG_DBG("Start Hello sample");
+
+	if (IS_ENABLED(CONFIG_GOLIOTH_SAMPLE_WIFI)) {
```

Comment:
Surprised to see this here, since it's an Ethernet sample. Any reason this is needed?

Reply:
This was leftover from our Hello sample, I've removed it from this repo, thanks!